compiler+runtime/src/cpp/main.cpp

```diff
@@ -119,6 +119,7 @@ namespace jank
       throw std::runtime_error{ "Not yet implemented: REPL server" };
     }
 
+    if(opts.target_module != "clojure.core")
```

Comment:
Why is this necessary? If we try to `load_module` `clojure.core` twice, it should already be loaded the second time and do nothing, right?